sdk/netapp/arm-netapp/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
     
+## 22.0.0 (2025-07-15)
+    
+### Features Added
+  - Interface SubscriptionQuotaItemList has a new optional parameter nextLink
+  - Added Type Alias NetAppResourceQuotaLimitsListNextResponse
+  - Enum KnownReplicationSchedule has a new value TenMinutely
+### Breaking Changes
+  - Enum KnownReplicationSchedule no longer has value "10Minutely"
```

Comment:
i remember this is not expected, because we have generation isuse so we need to manually update this?